benchmarks/bigint_conversion.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import arkouda as ak
+import time
+
+
+def time_bigint_conversion(N, trials, seed, max_bits):
```

Comment:
I'd change `N` to `N_per_locale` to match other benchmark (in the original test I wrote I was just being lazy and wanted to keep the code shorter, but I think we want it to scale with the number of nodes for the real benchmark)

Reply:
oh yeah! I think is the second time now i've modeled after the `array_transfer` benchmark and forgotten to do this